aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,194 @@
+  * Groups of related pages can be kept in their own namespace and directory.
+
+To get started with your first ASP.NET Core Razor Pages app, see <xref:tutorials/razor-pages/razor-pages-start>. For a complete overview of ASP.NET Core Razor Pages, its architecture and benefits, see: <xref:razor-pages/index>.
+
+### ASP.NET Core MVC
+
+ASP.NET MVC renders UI on the server and uses a Model-View-Controller (MVC) architectural pattern. The MVC pattern separates an app into three main groups of components: Models, Views, and Controllers. User requests are routed to a controller. The controller is responsible for working with the model to perform user actions or retrieve results of queries. The controller chooses the view to display to the user, and provides it with any model data it requires. Support for Razor Pages is built on ASP.NET Core MVC.
+
+MVC benefits, in addition to the server rendering benefits:
+
+* Some developers consider it easier to scale the web UI app in terms of complexity. They feel it's easier to code, debug, and test the model, view, or controller when it has a single job.
+* Clear [separation of concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns) for maximum flexibility.
+* The Model-View-Controller separation of responsibilities ensures that the business model can evolve without being tightly coupled to low-level implementation details.
+
+To get started with  ASP.NET Core MVC, see <xref:tutorials/first-mvc-app/start-mvc>. For an overview of ASP.NET Core MVC's architecture and benefits, see <xref:mvc/overview>.
+
+## Choose a predominantly client rendered ASP.NET Core solution
+
+The following section briefly explains the ASP.NET Core web UI client rendered models available and provides links to get started.
+
+### Blazor
+
+Blazor apps are composed of Razor components: segments of reusable, web UI implemented using C#, HTML, and CSS. Both client and server code are written in C#, allowing shared code and libraries. Razor components can be rendered or prerendered from views and pages.
+
+Benefits of Razor components:
+
+* Build interactive web UIs using C# rather than JavaScript. Using the same language for front-end and back-end code can:
+  * Accelerate app development.
+  * Reduce build pipeline complexity.
+  * Simplify maintenance.
+  * Leverage the existing .NET ecosystem of [.NET libraries](/dotnet/standard/class-libraries).
+  * Let developers understand and work on both client-side and server-side code.
+* Create reusable, sharable UI components.
+* Quickly get productive with Blazor reusable UI components from [top component vendors](https://blazor.net).
+* Work with all modern web browsers, including mobile browsers. Blazor uses open web standards without plug-ins or code transpilation.
+
+For more information, see <xref:blazor/components/prerendering-and-integration>.
+
+#### Blazor hosting options:
```

Comment:
Instead of duplicating this content here, can we just say that Blazor components can be hosted in different ways to take advantage of server and client rendering and then point to https://docs.microsoft.com/aspnet/core/blazor/hosting-models for a discussion of the different hosting models?